packages/import/src/cli.ts

```diff
@@ -97,7 +97,8 @@ inquirer.registerPrompt("filePath", require("inquirer-file-path"));
     }
 
     if (importer) {
-      await importIssues(importAnswers.linearApiKey, importer);
+      const apiUrl = process.argv.includes("--local") ? "https://local.linear.dev:8090/graphql/" : undefined;
```

Comment:
nit: I would have moved ``https://local.linear.dev:8090/graphql/` out of the actual cli code and have it passed as a variable from the npm script